rules/external_binaries/def.bzl

```diff
@@ -56,15 +76,17 @@ _external_binary = repository_rule(
             doc = "Version of the binary",
             mandatory = False,
         ),
+        "strip_prefix": attr.string_dict(
+            allow_empty = True,
+            doc = "Directory prefixex to strip from the extracted files",
```

Comment:
nitpick: small typo ~prefixex~ prefixes

Reply:
Oh yeah, I caught that as well when I moved it to its own repo, so it's fixed there already.